File: project/jinsoo-onboarding/service/survey/src/main/java/survey/survey/SurveyApplication.java
package survey.survey;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@EntityScan(basePackages = "survey")
@EnableJpaRepositories(basePackages = "survey")
@SpringBootApplication
public class SurveyApplication {
    public static void main(String[] args) {
        SpringApplication.run(SurveyApplication.class, args);
    }
}

File: project/hajin-onboarding/src/main/kotlin/com/innercircle/survey/SurveyApplication.kt
package com.innercircle.survey

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class SurveyApplication

fun main(args: Array<String>) {
    runApplication<SurveyApplication>(*args)
}